test/deploy-pipeline-mkp-cli.sh

```diff
@@ -14,91 +14,10 @@
-sed -ri 's/publishedVersion:.*/publishedVersion: '"$GCR_IMAGE_TAG"'/' schema.yaml
-sed -ri 's/version:.*/version: '"$GCR_IMAGE_TAG"'/' ./chart/kubeflow-pipelines/templates/application.yaml
-
-# Build new deployer
-export REGISTRY=gcr.io/$(gcloud config get-value project | tr ':' '/')
-export APP_NAME=$COMMIT_SHA
-export DEPLOYER_NAME=$REGISTRY/$APP_NAME/deployer
-
-sudo docker build --tag $DEPLOYER_NAME -f deployer/Dockerfile .
-
-sudo docker push $DEPLOYER_NAME:$GCR_IMAGE_TAG
-
-# Copy rest images and rename current images
-export MKP_VERSION=$GCR_IMAGE_TAG
-export GCR_FOLDER=$REGISTRY/$APP_NAME
-
-echo MKP_VERSION:$MKP_VERSION, GCR_FOLDER:$GCR_FOLDER
-
-export ARGO_VERSION=v2.3.0-license-compliance
-export CLOUDSQL_PROXY_VERSION=1.14
-export MLMD_SERVER_VERSION=0.21.1
-export MLMD_ENVOY_VERSION=initial
-export MINIO_VERSION=RELEASE.2019-08-14T20-37-41Z-license-compliance
-export MYSQL_VERSION=5.6
-
-gcloud container images add-tag --quiet gcr.io/ml-pipeline/argoexec:$ARGO_VERSION $GCR_FOLDER/argoexecutor:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/ml-pipeline/workflow-controller:$ARGO_VERSION $GCR_FOLDER/argoworkflowcontroller:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/cloudsql-docker/gce-proxy:$CLOUDSQL_PROXY_VERSION $GCR_FOLDER/cloudsqlproxy:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/tfx-oss-public/ml_metadata_store_server:$MLMD_SERVER_VERSION $GCR_FOLDER/metadataserver:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/ml-pipeline/envoy:$MLMD_ENVOY_VERSION $GCR_FOLDER/metadataenvoy:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/ml-pipeline/minio:$MINIO_VERSION $GCR_FOLDER/minio:$MKP_VERSION
-gcloud container images add-tag --quiet gcr.io/ml-pipeline/mysql:$MYSQL_VERSION $GCR_FOLDER/mysql:$MKP_VERSION
-
-gcloud container images add-tag --quiet $GCR_FOLDER/api-server:$MKP_VERSION $GCR_FOLDER/apiserver:$MKP_VERSION
-gcloud container images add-tag --quiet $GCR_FOLDER/inverse-proxy-agent:$MKP_VERSION $GCR_FOLDER/proxyagent:$MKP_VERSION
-gcloud container images add-tag --quiet $GCR_FOLDER/viewer-crd-controller:$MKP_VERSION $GCR_FOLDER/viewercrd:$MKP_VERSION
-gcloud container images add-tag --quiet $GCR_FOLDER/visualization-server:$MKP_VERSION $GCR_FOLDER/visualizationserver:$MKP_VERSION
-gcloud container images add-tag --quiet $GCR_FOLDER/deployer:$GCR_IMAGE_TAG $GCR_FOLDER/pipelines-test/deployer:$GCR_IMAGE_TAG
-
-# Run install script
-mpdev scripts/install  --deployer=$DEPLOYER_NAME:$GCR_IMAGE_TAG   --parameters='{"name": "'$APP_INSTANCE_NAME'", "namespace": "'$NAMESPACE'"}'
-
-echo "Status of pods after mpdev install"
-kubectl get pods -n ${NAMESPACE}
-
-popd
-
-# Waiting for the KFP resources are ready. TODO: verification of KFP resources
-sleep 60
+# sync trigger to avoid wait
+gcloud builds submit --config=test/cloudbuild/mkp_verify.yaml --substitutions=_MM_VERSION="$VERSION",COMMIT_SHA="$COMMIT_SHA" --project=ml-pipeline-test
+return $?
```

Comment:
I wonder why we don't set it as `set -ex` and remove this one?

Reply:
done